src/graph_impl/mod.rs

```diff
@@ -262,6 +262,9 @@ impl<E, Ix: IndexType> Edge<E, Ix>
 /// - Edge type `Ty` that determines whether the graph edges are directed or undirected.
 /// - Index type `Ix`, which determines the maximum size of the graph.
 ///
+/// The `Graph` is a regular rust collection and is `Send` and `Sync` (as long
```

Comment:
Please capitalize "Rust" here.

Reply:
Ah, I should have caught that when I copy/pasted the code.  Thanks will fix